src/atomate2/common/jobs/phonons.py

```diff
@@ -246,7 +246,8 @@ def generate_frequencies_eigenvectors(
     )
 
 
-@job(data=["forces", "displaced_structures"])
+# @job(data=["forces", "displaced_structures"])
```

Comment:
Just a reminder: this is really important for large structures. Just saw this change by chance.